TreeTracker/Resources/Localizable.strings

```diff
@@ -99,3 +100,19 @@
 "Settings.PhotoQuality.Option.Medium.Info" = "Medium resolution, medium data usage.";
 "Settings.PhotoQuality.Option.High.Title" = "High";
 "Settings.PhotoQuality.Option.High.Info" = "High resolution, high data usage.";
+
+//ChatList Screen
+"ChatList.NoMessagesLabel.Text" = "No messages, Yet.";
+
+//Messages Screen
+"Messages.Title" = "Admin";
+"Messages.InputTextView.PlaceHolder" = "Click to write a message";
+
+//Announce Screen
+"Announce.Title" = "Announce";
```

Comment:
I think "Announcement" would be better

Reply:
Sure! Do you want me to change just the string or all files name of that type?

Reply:
Just the string is fine 👍